Project.toml

```
@@ -1,7 +1,7 @@
 name = "GradedArrays"
 uuid = "bc96ca6e-b7c8-4bb6-888e-c93f838762c2"
 authors = ["ITensor developers <support@itensor.org> and contributors"]
-version = "0.4.7"
+version = "0.4.8"
```

```suggestion
version = "0.4.9"
```
I merged and registered GradedArrays.jl v0.4.9 in #42 so this PR will have to bump to v0.4.9.